packages/react/src/overrides.d.ts

```diff
@@ -229,6 +229,7 @@ import {
 } from './components/Table';
 import { TextFieldGroupClassKey, TextFieldGroupProps } from './components/TextFieldGroup';
 import { buttonMixin, listItemMixin } from './theming/mixins';
+import { TimelineClassKey, TimelineProps } from './components';
```

Comment:
```suggestion
import { TimelineClassKey, TimelineProps } from './components/Timeline';
```
И `TimelineItem` добавить